packages/cli/src/WebhookHelpers.ts

```diff
@@ -246,6 +246,22 @@ export function getWorkflowWebhooksBasic(workflow: Workflow): IWebhookData[] {
 			didSendResponse = true;
 		}
 
+		if (responseMode === 'noBodyResponse' && didSendResponse === false) {
+			if (webhookResultData.webhookResponse !== undefined) {
```

Comment:
Could you please remove this `if` block?

The `webhookResponse` property is only set by the `Webhook.node.ts` when the `responseData` property exists, and it's only then possible to set it when `Response Mode` is set to `On Received`.

You can leave the code in the `else` block only.

Thanks!